docs/badge.md

```diff
@@ -142,3 +163,63 @@ Custom component to replace the badge outer component
 |          Type          |                  Default                   |
 | :--------------------: | :----------------------------------------: |
 | React Native Component | View, if `onPress` then `TouchableOpacity` |
+
+---
+
+## Props for withBadge
+
+* [`value`](#value)
+* [`options`](#options)
+
+---
+
+## Reference for withBadge
+
+### `value`
+Text value to be displayed by badge, defaults to empty
+
+|                                              Type                                       | Default |
+| :-------------------------------------------------------------------------------------: | :-----: |
+| String OR Number OR React Native Component OR Function, which returns one of the former |  none   |
+
+---
+
+### `options`
+
+> Also receives all
+> [Badge](https://facebook.github.io/react-native/docs/badge#props) props.
+
+Object with the following (optional) keys:
+
+#### `bottom`
+
```

Comment:
Hey @janhesters we're just gonna go forward and merge it.

Can you just update the default values to match the code here?

Reply:
Done 👍